--- originals/gen.c	2025-01-21 09:14:08.362347782 -0500
+++ gen.c	2025-01-21 09:14:33.390478815 -0500
@@ -320,7 +320,7 @@
 				perror("dup2");
 			close(pipedes[0]); /* close reading pipe end */
 			execlp("sox", "sox", 
-			       "-t", "raw",  "-s", "-2", "-r", srate, "-",
+			       "-t", "raw", "-e","signed-integer", "-b", "16","-r",srate, "-",
 			       "-t", type, fname,
 			       NULL);
 			perror("execlp");
